pkg/setup/setup.go

```diff
@@ -366,6 +366,12 @@ func (s *Setup) installPackageServicesPlugins(wg *sync.WaitGroup, httpClient *ht
 		return
 	}
 
+	// Install all package CLIs when the env var is present.
+	installPackageCLIs, _ := s.envLookup("DCOS_CLI_EXPERIMENTAL_AUTOINSTALL_PACKAGE_CLIS")
```

Comment:
There is a call to list packages happening before this check, could this be moved on top of the method instead?